song-server/src/main/java/bio/overture/song/server/service/StorageService.java

```diff
@@ -58,18 +58,19 @@
   @NonNull private final RetryTemplate retryTemplate;
   @NonNull private final String storageUrl;
   @NonNull private final ValidationService validationService;
+  @NonNull private final String scoreAuthorizationHeader;
 
   @SneakyThrows
   public boolean isObjectExist(@NonNull String accessToken, @NonNull String objectId) {
     return doGetBoolean(accessToken, getObjectExistsUrl(objectId));
   }
 
   @SneakyThrows
-  public StorageObject downloadObject(@NonNull String accessToken, @NonNull String objectId){
-    val objectExists = isObjectExist(accessToken, objectId);
+  public StorageObject downloadObject(@NonNull String objectId){
+    val objectExists = isObjectExist(scoreAuthorizationHeader, objectId);
     checkServer(objectExists,getClass(), STORAGE_OBJECT_NOT_FOUND,
         "The object with objectId '%s' does not exist in the storage server", objectId);
-    return convertStorageDownloadResponse(objectId, getStorageDownloadResponse(accessToken, objectId));
+    return convertStorageDownloadResponse(objectId, getStorageDownloadResponse(scoreAuthorizationHeader, objectId));
```

Comment:
likewise, make `getStorageDownloadResponse` internally dependent on the member variable `scoreAuthorizationHeader`, just like for `isObjectExist`

Reply:
essentially, no non-static method should use `scoreAuthorizationHeader` as a method parameter